workloads/router-perf-v2/common.sh

```diff
@@ -220,3 +220,5 @@ test_routes(){
     done <<< $(oc get route -n http-scale-${termination} --no-headers | awk '{print $2}')
   done
 }
+
+python3 -m pip install -r requirements.txt | grep -v 'already satisfied'
```

Comment:
Please add newline at the end of file.

Reply:
@kedark3 Thank you for the comment! I added a newline and squashed the commits.

Reply:
@kedark3 Will you merge the PR?